Neo4j.Driver/Neo4j.Driver/Internal/Extensions/CollectionExtensions.cs

```diff
@@ -49,7 +49,7 @@ public static T GetValue<T>(this IDictionary<string, object> dict, string key, T
 
         public static string ToContentString<K, V>(this IDictionary<K, V> dict)
         {
-            var output = dict.Select(item => $"{{{item.Key}, {item.Value}}}");
+            var output = dict.Select(item => $"{{{item.Key}, {item.Value.ValueToString()}}}");
```

Comment:
Statistics will be reported in a dictionary. It depends on how the end user would like to print. 

The real way of consuming statistics in my mind was that we will give the dictionary to a graphic server and show us all statistics in real time (every a few sec). I would suggest to not bother too much of how we internally print them out now. We are not really relying on console printout in our tests anyway.